internal/imports/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta name="go-import" content="gocloud.dev git https://github.com/google/go-cloud">
+    <meta name="go-source" content="gocloud.dev https://github.com/google/go-cloud https://github.com/google/go-cloud/tree/master{/dir} https://github.com/google/go-cloud/tree/master{/dir}/{file}#L{line}">
+    <meta http-equiv="refresh" content="0; url=https://github.com/google/go-cloud">
+  </head>
+  <body>
+    Redirecting you to our GitHub home page.
```

Comment:
Could redirects fail? (maybe some browser configuration, IIRC browsers can disable this). What I've commonly seen done is somethiing like:

You will be redirected to the project's Github page shortly. If that doesn't happen, [click here][link]

Reply:
Done.